packages/patrol_cli/lib/src/ios/ios_test_backend.dart

```diff
@@ -140,12 +136,6 @@ class IOSTestBackend {
       final subject = '${options.description} on ${device.description}';
       final task = _logger.task('Running $subject');
 
-      Process? iosDeployProcess;
```

Comment:
Calling [XCUIApplication.launch()](https://developer.apple.com/documentation/xctest/xcuiapp
lication/1500467-launch) with the debugger attached causes really nasty crashes.